internal/analytics/history.go

```
@@ -0,0 +1,271 @@
+		}
+		tables = append(tables, name)
+	}
+	if err := rows.Err(); err != nil {
+		return nil, fmt.Errorf("iterate tables in %s: %w", absPath, err)
+	}
+
+	counts := make(map[string]int64, len(tables))
+	for _, tbl := range tables {
+		var n int64
+		// Double-quote the identifier; table names come from sqlite_master.
+		if err := db.QueryRow(`SELECT COUNT(*) FROM "` + tbl + `"`).Scan(&n); err != nil {
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).QueryRow must not be called. use (*database/sql.DB).QueryRowContext

(noctx)